### DISCUSSION TITLE
Vulnerabilities

### DISCUSSION THREAD

**Discussion by dharsani90** [community/community#61199] [Code Security]
### Select Topic Area

General

### Body

Is there any possibility to stop publishing vulnerabilities publicly?

**Comment by siennaphia** [comment]
Yes, GitHub Advanced Security provides options to manage the visibility of security vulnerabilities. By default, GitHub Advanced Security publishes security vulnerabilities publicly to help the open-source community identify and fix vulnerabilities. However, you can configure the visibility settings to stop publishing vulnerabilities publicly.



1. You can disable or enable the 'Dependency graph' and 'Dependabot alerts' in the repository settings. If you disable these features, GitHub will stop scanning for vulnerable dependencies in the repository, and therefore, will stop publishing alerts [docs.github.com](https://docs.github.com/en/free-pro-team@latest/github/managing-security-vulnerabilities/about-alerts-for-vulnerable-dependencies).

Navigate to your repository on GitHub -> Settings -> Security & analysis -> Dependency graph and Dependabot alerts

2. You can control who has access to security advisories in your repository. By default, security advisories are visible to people with read access to the repository. However, you can change this setting to limit the visibility to only people with write access.

Navigate to your repository on GitHub -> Security -> Security policy -> Edit
